code/modules/surgery/organs/internal/lungs/_lungs.dm

```diff
@@ -47,6 +47,8 @@
 	var/safe_plasma_min = 0
 	///How much breath partial pressure is a safe amount of plasma. 0 means that we are immune to plasma.
 	var/safe_plasma_max = 0.05
+	///The required partial pressure of water_vapor for not suffocating.
+	var/safe_water_level = 16 // same as oxygen
```

Comment:
You need to add
```
if(safe_water_level)
	add_gas_reaction(/datum/gas/water_vapor, always = PROC_REF(breathe_water))
```
To `/lungs/Initialize`.

And by default, `safe_water_level` should be 0.